hydra-node/src/Hydra/Options.hs

```diff
@@ -855,12 +855,19 @@ persistenceDirParser =
 persistenceRotateAfterParser :: Parser Natural
```

Comment:
Could: use `Positive` (https://hackage-content.haskell.org/package/QuickCheck-2.16.0.0/docs/Test-QuickCheck.html#t:Positive) or create a new type with a smart constructor

```haskell
newtype RotateAfter = UnsafeRotateAfter Natural

mkRotateAfter :: MonadFail m => Natural -> m RotateAfter

instance Read where
  readPrec = fmap mkRotateAfter readPrec
```

Not too sure whether `Read` is a good instance to define at all. Maybe just use `maybeReader (readMaybe <&> mkRotateAfter)` or so.